modules/ROOT/pages/ios_app/faq.adoc

```diff
@@ -0,0 +1,61 @@
+= FAQ
+:toc: right
+:description: This page provides frequently asked questions about branding the iOS app. 
+
+== General Information
+
+See the https://developer.apple.com/app-store/review/[App Review] description from Apple for more details on reviewing an iOS app.
+
+== Cryptography and Export Classifies Exemptions
+
+The product does not contain cryptography and does therefore need anyexport exemptions. Although the iOS app is ready to connect via SSL, this does not imply that the iOS app includes any cryptography.
+
+== Utilisation of Document Picker and File Provider Extensions
+
+The iOS app takes advantage of the `File Provider` extensions so that those apps that act as `Document Picker` may access to the ownCloud data, edit it and then changes are automatically uploaded back to the ownCloud instance.
```

Comment:
```suggestion
The iOS app takes advantage of the `File Provider` extensions so that those apps that act as `Document Picker` may access the ownCloud data, edit it and then changes are automatically uploaded back to the ownCloud instance.
```